### PR TITLE
feat: pay 0 amount invoices in LND

### PR DESCRIPTION
**Pull request by im-adithya**
Adds 0 amount invoices for LND

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
     "prepare": "cd .. && husky frontend/.husky"
   },
   "dependencies": {
-    "@getalby/lightning-tools": "^5.1.2",
     "@getalby/bitcoin-connect-react": "^3.6.2",
+    "@getalby/lightning-tools": "5.1.2",
```
Comment (rolznz) [inline frontend/package.json:24]: Why this change?

Comment (im-adithya) [reply frontend/package.json:24]: I did yarn add @getalby/lightning-tools, and it corrected this, it should be in alphabetical order right?

Comment (rolznz) [reply frontend/package.json:24]: it changed it to the exact version, and this was already in the PR (You probably should have just did yarn install?), I think the change should be undone

Comment (im-adithya) [reply frontend/package.json:24]: Yeah, but isn't it the convention? To have the packages in sorted order? https://eslint.org/docs/latest/contribute/package-json-conventions#order

Comment (rolznz) [reply frontend/package.json:24]: The order is not the only change, you also changed the version from "^5.1.2" to "5.1.2"
